hbase_sql/pom.xml

```diff
@@ -774,6 +774,47 @@
             <plugin>
                 <artifactId>maven-antrun-plugin</artifactId>
                 <executions>
+                    <execution>
+                        <id>############### clean jars ###############</id>
```

Comment:
is this the right name for this step? 

perhaps 'clean standalone jars' or just 'standalone build' ?